rmgpy/rmg/main.py

```diff
@@ -1268,7 +1268,16 @@ def make_seed_mech(self, first_time=False):
             if os.path.exists(seed_dir):  # This is a seed from a previous RMG run. Delete it
                 shutil.rmtree(seed_dir)
         else:  # This is a seed from the previous iteration. Move it to a temporary directory in case we run into errors
-            os.rename(seed_dir, os.path.join(temp_seed_dir))
+            try:
+                os.rename(seed_dir, temp_seed_dir)
```

Comment:
Have you considered the tempdirectory option? I am curious if it would have advantages in this case. https://docs.python.org/3/library/tempfile.html#tempfile.tempdir

Reply:
I had not, thanks for the suggestion! It looks interesting, but I am not sure if I can move existing files inside of a temporary directory. Since we already have the previous seed files in a non-temporary place, it should be much faster to rename/move the files (which does not actually move the data on the disk) rather than copying. So if we can move these files into the temporary directory without making a copy then it might be interesting to implement. The only added advantage I think is just the automatic cleanup, but it would be cleaner code.